src/boot/stylesheets/overlay-bars.scss

```diff
@@ -9,10 +9,6 @@
 	background-color: $alert-red;
 	color: $white;
 
-  @media only screen and (min-width: 800px) {
-		width: 400px;
```

Comment:
I believe that's a separate issue; you need to click the Back button to notice it on the main list view – it's the same in prod right now. I meant to ask @drw158 if there was any reason that was a deliberate choice, or just an oversight for the narrow single view.